### PR TITLE
Fresh test! 

### PR DESCRIPTION
**Pull request by colleenmcginnis**
This is yet another test replacing #179.

This is running `docs-preview / doc-preview-pr (pull_request_target)` but not `buildkite/docs-build-pr`.

### CODE REVIEW

**Comment by colleenmcginnis** [comment]
Ok we got a new error at least!! https://buildkite.com/elastic/docs-build-pr/builds/182866#01930292-6816-4d1b-a7ea-749aa5132500

File: serverless/pages/action-connectors.asciidoc [hunk 1]
```diff
@@ -4,6 +4,8 @@
 // :description: Configure connections to third party systems for use in cases and rules.
 // :keywords: serverless
 
+hello world !!!!
```
Comment (leemthompo) [inline serverless/pages/action-connectors.asciidoc:7]:
```suggestion
hello world !!!!!
```

Comment (colleenmcginnis) [reply serverless/pages/action-connectors.asciidoc]: Came here to test another commit.... so thanks !!!!!

**Comment by colleenmcginnis** [comment]
@bmorelli25 it looks like:

* `buildkite/docs-build-pr` is showing up in the list of checks 🎉 
* The build is still failing with the error `fatal: reference repository '/opt/git-mirrors/elastic-docs-content' is not a local repository.` 🪞 
* Can you remove the `publish / doc builder` from the branch settings? 🗑️  (We added this yesterday whilst troubleshooting, but I don't have access to those settings.)

**Comment by bmorelli25** [comment]
run docs-build

**Comment by bmorelli25** [comment]
It's working!!!!

**Comment by colleenmcginnis** [comment]
run docs-build

**Comment by colleenmcginnis** [comment]
run docs-build

**Comment by colleenmcginnis** [comment]
IT WORKED FOR REAL !!!!! https://docs-content_bk_180.docs-preview.app.elstc.co/guide/en/serverless/current/action-connectors.html

image

**Comment by leemthompo** [comment]
GE2lkmNXUAAJora

**Pull request closed without merging**